tests/checks/common.py

```diff
@@ -300,7 +300,7 @@ def coverage_report(self):
             untested_events=pformat(untested_events),
         ))
 
-        if os.getenv('COVERAGE'):
+        if not os.getenv('NO_COVERAGE'):
```

Comment:
nitpick: setting `NO_COVERAGE=false` would still disable the enforcement of the coverage :trollface: